xain/benchmark/aggregation/final_task_accuracies.py

```diff
@@ -112,9 +112,9 @@ def aggregate() -> str:
 
     fpath = plot(
         data,
-        title="Max achieved accuracy for unitary and federated learning",
-        xlabel="partitioning grade",
-        ylabel="accuracy",
+        title="",  # TODO
```

Comment:
Is this something we want to do in this pr?

Reply:
No, it's not for this PR. I left the comment in there to follow up on this - ideally we want to include a way in our benchmark "DSL" to configure this per group or per task